src/textual/widgets/_placeholder.py

```diff
@@ -61,10 +62,10 @@ class Placeholder(Widget):
         overflow: hidden;
         color: $text;
     }
-
     Placeholder.-text {
         padding: 1;
     }
+
```

Comment:
Why add a hanging empty line? Was this an artifact of some automated tool?

I've seen many empty lines hanging elsewhere, e.g., at the end of docstrings, and I don't get where they come from. 🤷

Reply:
Probably just the result of an edit that was backed out of. Black will add new lines in some places.